packages/rollup/rollup_bundle.bzl

```diff
@@ -312,7 +312,8 @@ def _rollup_bundle(ctx):
 
     stamp = ctx.attr.stamp[StampSettingInfo].value
 
-    config = ctx.actions.declare_file("_%s.rollup_config.js" % ctx.label.name)
+    config_extension = ctx.file.config_file.extension
+    config = ctx.actions.declare_file("_%s.rollup_config.%s" % (ctx.label.name, config_extension))
```

Comment:
Can we just change it to mjs 100% of the time?

Reply:
I don't think so, since configs may also be CommonJS. If you want to enforce ESM rollup configs, that works, but it's no cost shifting that responsibility of supporting ESM or CJS to rollup itself.